cves/CVE-2017-17081.yml

```diff
@@ -131,7 +144,15 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    VCC1 Date:  Tue Apr 4 09:23:45 2006 +0000
+    Fix Date:   Mon Oct 9 00:32:30 2017 +0200
+    Lifetime:   ~11 years
+
+    The dsputil.c refactored... Aha. I just realized I've accidentally done a third.
```

Comment:
```suggestion
    The dsputil.c refactored.
```

Reply:
Perhaps something to state it's lacking in description? If not I can still fill in the rest of the description, but I may not get to it until this weekend.